sklearn/mixture/dpgmm.py

```diff
@@ -584,6 +593,12 @@ def fit(self, X):
 
         self._set_weights()
 
+        # Creat covariance which is used in the sampling function
+        if self.covariance_type in ['diag', 'spherical']:
+	    self.covars_ = [np.reciprocal(cov) for cov in self.precs_]
+	else:
+	    self.covars_ = [pinvh(c) for c in self._get_precisions()]
+
```

Comment:
You're using tabs here. Please use spaces only. Python 3 considers mixing tabs and spaces a syntax error.